src/cmds/qmgr.c

```diff
@@ -110,6 +110,7 @@
 #define QMGR_TIMEOUT 900 /* qmgr connection timeout set to 15 min */
 time_t start_time = 0;
 time_t check_time = 0;
+int num_connected_svrs = 0;
```

Comment:
Why do you need this variable at the qmgr.c level. I mean, if the connections are managed by IFL transparently, do we need to track anything at the level of the clients (or commands)?

Besides, it appears that num_connected_svrs is not being used in qmgr.c?